sw/host/ot_certs/src/template/mod.rs

```diff
@@ -75,7 +75,9 @@ pub struct Certificate {
     pub subject_key_identifier: Value<Vec<u8>>,
     // X509 basic constraints extension, optional.
     pub basic_constraints: Option<BasicConstraints>,
-    /// X509 certificate extensions.
+    /// X509 Subject Alternative Name extension, optional.
+    pub subject_alt_name: Option<IndexMap<AttributeType, Value<String>>>,
```

Comment:
As discussed, I started with declaring this as `IndexMap<>`, but the hjson parser started failing on all templates where this field was not persent.

Reply:
You can remove the `Option` by diing:
```rust
#[serde(default)]
    subject_alt_name: IndexMap<...>,
```